README.md

```diff
@@ -298,6 +306,24 @@ console.log("will raise an exception if not in script_hashes.yml!")
 <% end %>
 ```
 
+### Public Key Pins
+
+```
+config.hpkp = {
+  max_age: 60.days.to_i,   # max_age is a required parameter
+  include_subdomains: true, # whether or not to apply pins to subdomains
+  # Per the spec, SHA256 hashes are the only currently supported format.
+  pins: [
+    {sha256: 'b5bb9d8014a0f9b1d61e21e796d78dccdf1352f23cd32812f4850b878ae4944c'},
+    {sha256: '73a2c64f9545172c1195efb6616ca5f7afd1df6f245407cafb90de3998a1c97f'}
+  ],
+  enforce: true,            # defaults to false (report-only mode)
+  report_uri: '//example.com/uri-directive',
```

Comment:
You may want to add a note and/or a warning that `report_uri` and `Public-Key-Pins-Report-Only` have not yet been implemented yet in Chrome: https://code.google.com/p/chromium/issues/detail?id=445793#c2 (I'm not sure of the status in Firefox). Similarly, it may be good to document/warn that setting a `report_uri` to the same domain as the hosting application can lead to unexpected behavior because, as noted in http://tools.ietf.org/html/draft-ietf-websec-key-pinning-21#section-2.1.4:

> If the scheme in
   the report-uri is one that uses TLS (e.g.  HTTPS), UAs MUST perform
   Pinning Validation when the host in the report-uri is a Known Pinned
   Host; similarly, UAs MUST apply HSTS if the host in the report-uri is
   a Known HSTS Host.

So, if you have a pin failure you will not be able to send a report back to the hosting application via `https`. Moreover, if the site is HSTS you won't be able to send back a report of `https` or `http` (because of HSTS).